Tests.props

```diff
@@ -3,8 +3,7 @@
 
   <PropertyGroup>
     <SingleTestPlatform Condition="'$(SingleTestPlatform)' == ''">false</SingleTestPlatform>
-    <NoWarn>$(NoWarn);IDE0017</NoWarn>
-    <IsPackable>false</IsPackable>
```

Comment:
This `<IsPackable>false</IsPackable>` line must be reverted

Reply:
`IsPackable` is false by default for tests as it was before.

Reply:
Only when `IsPackable` is not set in `Directory.Build.props`.